operations/service_web.go

```diff
@@ -104,6 +105,10 @@ func startWebService() cli.Command {
 				close(adminWait)
 			}()
 
+			if c.Bool(graphqlFeatureFlag) {
+				graphql.StartServer()
+			}
```

Comment:
I think this has to go away so that the graphql service is hosted on the same port as the other services (and the handler change above will handle that. 

Having said that, and just for your own edification there are a couple of flaws here: 
- line 109 blocks and so none of the code after this point will run, ever. 
- the contexts that are passed into every other function in combination with the the process on line 113 (gracefulShutdownForSigterm) means that when the service terminates, the service won't wait for in progress requests to complete, and will just terminate hard. That's bad. If you ran 109 in a go statement, then the existing clean-shutdown code would work, but the graphql service would just have hard termination after everything gracefully terminates, which would be better but still not good for your case.